src/Jackalope/Transport/Jackrabbit/JackrabbitClientInterface.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Jackalope\Transport\Jackrabbit;
+
+use Jackalope\Transport\LockingInterface;
+use Jackalope\Transport\NodeTypeCndManagementInterface;
+use Jackalope\Transport\ObservationInterface;
+use Jackalope\Transport\PermissionInterface;
+use Jackalope\Transport\QueryInterface as QueryTransport;
+use Jackalope\Transport\VersioningInterface;
+use Jackalope\Transport\WorkspaceManagementInterface;
+use Jackalope\Transport\WritingInterface;
+
+/**
+ * Internal interface for jackrabbit client.
```

Comment:
Maybe "Collect all interfaces the jackrabbit client implements and define the additional jackrabbit specific methods."

and then on a new line `@internal`